client/app/scripts/superdesk-highlights/views/mark_highlights_dropdown_directive.html

```diff
@@ -1,7 +1,13 @@
 <ul>
 	<li ng-if="highlights.length" ng-repeat="h in highlights"> 
-	    <button ng-if="isMarked(h)" ng-disabled="true" option="{{ h.name | uppercase  }}" ng-click="markItem(h);">{{ h.name | uppercase  }}</button>
-	    <button ng-if="!isMarked(h)" option="{{ h.name | uppercase  }}" ng-click="markItem(h);">{{ h.name | uppercase  }}</button>
+	    <button ng-if="isMarked(h)" ng-disabled="true" option="{{ h.name  }}" ng-click="markItem(h);">
```

Comment:
instead of two identical blocks, this can be handled by ng-disabled="isMarked(h)"